twarep/electrifly
# UI updates

## Checklist

## What I Did
- Changed the graphing interface to make it smoother and less crowded with multiple input channels.
- Put the custom graph on another section and changed the input parameters. This will make it better to view the data.
- Changed the input format from `input_select` to `input_selectize` for better UI visage.
- Made changes to the date selection field by making it more readable and adding the timings for each date.

## How I Did it?
- By using existing packages and a bit of tweaking around the functions and the UI. It is much more seemless now.

## How to Test?
- It will be all manual tests. Please run `app.py` and go to the `Data Visuals` section. From there play around with the dates and the different sub sections. Please test that you can see the new data loading in the graphs and tables. Play around with the custom graph to see if there are any discrepancies (change date, graph type, x and y variables).
